packages/story-editor/src/components/library/panes/media/local/hotlink/useInsert.js

```diff
@@ -54,6 +60,23 @@ function getErrorMessage(code, description) {
   }
 }
 
+function seekVideo(video) {
```

Comment:
This can could be reused in `preloadVideo`.

Reply:
Yeah maybe, it's not super straightforward to reuse though because it just works off an existing video el. Not the most beautiful code :)

Reply:
You could move seekVideo function to the media package and export it. Then use seek video in preloadVideo.

Reply:
```
function preloadVideo(src) {
  const video = document.createElement('video');
  video.muted = true;
  video.crossOrigin = 'anonymous';
  video.preload = 'auto';

  return seekVideo(video);
}
```